resources/views/statistics.blade.php

```diff
@@ -0,0 +1,10 @@
+@extends('layouts.app')
+@section('content')
+    <div class="main-panel">
+        <div class="container-fluid">
+            <h4>{{ Auth::user()->firstname}}'s expenses</h4>
```

Comment:
Was this intended?
Maybe user's Statistics. Wdyt?
http://picpaste.com/pics/stats-VgwBBfIT.1490755551.PNG